### ISSUE TITLE
ALL videos in Music Category cannot download

### ISSUE THREAD

**Issue by Campiotti** [flagbug/YoutubeExtractor#268]
I tested this on multiple videos, every other video downloaded fine except every single video in the category "music", it throws some forbidden 403 error when I try to download it.

Error report (It's a german error report but you can reproduce the error by not having any try{}catch{} when executing the download of a youtube video in the music category):
YoutubeExtractor.YoutubeParseException ist aufgetreten.
  HResult=0x80131500
  Nachricht = Could not parse the Youtube page for URL http://youtube.com/watch?v=QK8mJJJvaes
This may be due to a change of the Youtube page structure.
Please report this bug at www.github.com/flagbug/YoutubeExtractor/issues
  Quelle = YoutubeExtractor
  Stapelüberwachung:
   bei YoutubeExtractor.DownloadUrlResolver.ThrowYoutubeParseException(Exception innerException, String videoUrl)
   bei YoutubeExtractor.DownloadUrlResolver.GetDownloadUrls(String videoUrl, Boolean decryptSignature)
   bei YTDownloaderCampiotti.YTDLCamp.DownloadVideo() in C:\Users\lucac\documents\visual studio 2017\Projects\YTDownloaderCampiotti\YTDownloaderCampiotti\Form1.cs: Zeile134
   bei YTDownloaderCampiotti.YTDLCamp.btnDownload_Click(Object sender, EventArgs e) in C:\Users\lucac\documents\visual studio 2017\Projects\YTDownloaderCampiotti\YTDownloaderCampiotti\Form1.cs: Zeile70
   bei System.Windows.Forms.Control.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnClick(EventArgs e)
   bei System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   bei System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   bei System.Windows.Forms.Control.WndProc(Message& m)
   bei System.Windows.Forms.ButtonBase.WndProc(Message& m)
   bei System.Windows.Forms.Button.WndProc(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   bei System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   bei System.Windows.Forms.NativeWindow.DebuggableCallback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
   bei System.Windows.Forms.UnsafeNativeMethods.DispatchMessageW(MSG& msg)
   bei System.Windows.Forms.Application.ComponentManager.System.Windows.Forms.UnsafeNativeMethods.IMsoComponentManager.FPushMessageLoop(IntPtr dwComponentID, Int32 reason, Int32 pvLoopData)
   bei System.Windows.Forms.Application.ThreadContext.RunMessageLoopInner(Int32 reason, ApplicationContext context)
   bei System.Windows.Forms.Application.ThreadContext.RunMessageLoop(Int32 reason, ApplicationContext context)
   bei System.Windows.Forms.Application.Run(Form mainForm)
   bei YTDownloaderCampiotti.Program.Main() in C:\Users\lucac\documents\visual studio 2017\Projects\YTDownloaderCampiotti\YTDownloaderCampiotti\Program.cs: Zeile19

Innere Ausnahme 1:
NotSupportedException: Das Ergebnis kann nicht für eine fehlgeschlagene Übereinstimmung aufgerufen werden.

**Comment by cooler-farmer** [comment]
This is because the decryption of the video doesn't work in the current version of YoutubeExtractor. I have working code for another project and when I find some time I'll implement it. 
In case you can do this yourself here is some code of a working decryption function:

https://gist.github.com/ef63f8d58dc58725ba15f59c1dd97aa5

Script Engine:
https://gist.github.com/24f551a7090e3653d5ee87ce9a043c3f
https://gist.github.com/a6d7c2f5de41c2316d68630aa9daa635
https://gist.github.com/e1acbb3ad39f4babf17033a7178b3173

**Comment by RickyDivjakovski** [comment]
Yeah this bug is killing me, i guess its got something to do with googles privacy updates

**Comment by Ricky310711** [comment]
This bug is doing my head in...
